libvirt/tests/src/backingchain/blockcopy/blockcopy_with_conventional_chain.py

```diff
@@ -53,7 +53,7 @@ def run_test():
         check_obj.check_mirror_exist(vm, target_disk, tmp_copy_path)
 
         if not utils_misc.wait_for(
-                lambda: libvirt.check_blockjob(vm_name, target_disk, "progress", "100"), 30):
+                lambda: libvirt.check_blockjob(vm_name, target_disk, "progress", "100.00"), 30):
```

Comment:
Do you need to check '100%' in other versions? what about updating like:
```suggestion
                lambda: libvirt.check_blockjob(vm_name, target_disk, "progress", "100(.00)?"), 30):
```

Reply:
Thanks yingshun, Good point ,Updated